iidm/iidm-api/src/main/javadoc/com/powsybl/iidm/network/doc-files/currentLimits.svg

```diff
@@ -1,68 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!--
-
-Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
-This Source Code Form is subject to the terms of the Mozilla Public
-License, v. 2.0. If a copy of the MPL was not distributed with this
-file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
--->
-
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-
 <svg
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:cc="http://creativecommons.org/ns#"
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:svg="http://www.w3.org/2000/svg"
    xmlns="http://www.w3.org/2000/svg"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="400"
-   height="400"
-   id="svg2"
    version="1.1"
-   inkscape:version="0.91 r13725"
```

Comment:
why these changes ? Is the svg broken ? on githubpreview and eyeofgnome the svg is empty. On inkscape, you see that the text is outside of the svg
![Capture d’écran de 2021-03-29 11-02-30](https://user-images.githubusercontent.com/89208/112813387-6fbf0200-907e-11eb-90aa-3d23f68b8f32.png)

Reply:
woops that's from my stash, nothing to do here